api/wasm/v0/cpp/proxy_wasm_externs.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * Proxy-WASM ABI.
+ */
+// NOLINT(namespace-envoy)
```

Comment:
Do we need some kind of namespacing for this file (regardless of Envoy)?

Reply:
I see, this is pure C. Should we be using `_` prefixes for all typs, e.g. enum results?

Reply:
I have never heard of _ prefixing of types.  Where have you seen that done?

Reply:
I was suggesting we do something like `Wasm_Foo` or `WasmFoo` for a `Foo` type that belong s to Wasm, given the lack of namespaces.

Reply:
Added Wasm prefix PTAL.